managers/flow_builder.py

```diff
@@ -95,8 +94,10 @@ def _build_int_source_flows(
     new_int_flow_tbl_2["flow"]["table_id"] = settings.INT_TABLE
     utils.set_table_group(new_int_flow_tbl_2)
 
-    # if intra-switch EVC, then output port should be the proxy
+    # if intra-switch EVC, then output port should be the dst UNI's source port
```

Comment:
I'll leave it as a draft until I cover with unit test, now that we know that the data plane is also working then it should be reasonable to do so.

Reply:
done on https://github.com/kytos-ng/telemetry_int/pull/50/commits/0a9113455cd95e07173530dd660ff5f8d4e2b65d